doorway-ui-components/src/page_components/listings/ZeroListingsItem.scss

```diff
@@ -0,0 +1,29 @@
+.zero-listings {
+    --margin: var(--bloom-s3);
+    --bottom-margin: var(--bloom-s6);
+    --border-color: var(--bloom-color-gray-450);
+    --item-padding-top: var(--bloom-s3);
+    --item-padding-bottom: var(--bloom-s6);
+    --description-padding-top: var(--bloom-s3);
+    --description-padding-bottom: var(--bloom-s6);
+  
+    margin: var(--margin);
+    margin-bottom: var(--bottom-margin);
+    padding-top: var(--item-padding-top);
+    padding-bottom: var(--item-padding-bottom);
+    list-style-type: none;
+    border-bottom: 1px solid var(--border-color);
+  }
```

Comment:
spacing on this file is wiiild lol. not a big deal but it seems like Prettier/autoformat may not be running in your uic? may want to look into that and fix on your IDE

Reply:
Thanks for pointing that out, Prettier had stopped working on save, I made this change https://github.com/microsoft/vscode/issues/108447 and that fixed it. Added a new commit with the formatting fixes.